app/db/index.ts

```
@@ -0,0 +1,110 @@
+import { Database } from '@nozbe/watermelondb';
+import SQLiteAdapter from '@nozbe/watermelondb/adapters/sqlite';
+import { mySchema } from './schema';
+import {
+  Profile,
+  Symbol,
```

Biome (1.9.4)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)